docs/angular/your-first-app.md

```diff
@@ -130,53 +138,87 @@ And voilà! Your Ionic app is now running in a web browser. Most of your app can
 
 ## Photo Gallery!!!
```

Comment:
Why do we have `!!!` here?

Reply:
🤷‍♀️ Whoever wrote it must have been excited for that section. I'll remove.

Reply:
https://github.com/ionic-team/ionic-docs/pull/4157/commits/c0d4ea830eee982a4e158b5ef855116255c2f902